CMakeLists.txt

```diff
@@ -48,6 +48,10 @@ if(BUILD_TESTING)
     include(AddGoogleTest)
     enable_testing ()
     add_subdirectory(sdk/core/azure-core/test/ut)
+    # Set `AZURE_CORE_DISABLE_JSON_TESTS` env var to disable Json tests
+    if(NOT DEFINED ENV{AZURE_CORE_DISABLE_JSON_TESTS})
```

Comment:
Let's make this test disabled by default since we don't expect to be modifying the json library in daily work, improves inner dev loop.